### PR TITLE
rG3kdLG9: Show the number of expiring certificates on dashboard

### PR DESCRIPTION
**Pull request by AmritSidhu**
Display the number of certificates expiring on the dashboard as per the prototype

### CODE REVIEW

File: app/helpers/user_journey_helper.rb [hunk 1]
```diff
@@ -26,4 +26,17 @@ def certificate_status(certificate)
...
+    expiring_certificates = 0
+    all_components.each do |component|
+      component.certificates.each do |certificate|
+        if certificate.expires_soon?
+          expiring_certificates += 1
+        end
+      end
+    end
+    expiring_certificates
+  end
```
Comment (brenetic) [inline app/helpers/user_journey_helper.rb]:
```suggestion
  def expiring_certificates_count(msa_components, sp_components)
    (msa_components + sp_components).map(:&certificates).map { |c| c.expires_soon? }.count
  end
```

just a thought 😛

**Comment by brenetic** [comment]
Can we have a bit of info in the PR message please

File: app/helpers/user_journey_helper.rb (hunk 1 above)
Comment (AmritSidhu) [reply app/helpers/user_journey_helper.rb]: Get undefined method `expires_soon?`

Comment (brenetic) [reply app/helpers/user_journey_helper.rb]: My bad. the initial map will bring back an empty array for any components that don't have any certs.

```
(msa_components + sp_components).map(&:certificates).flatten.select { |c| c.expires_soon? }.count
```

That'd work.

**Pull request merged**